doc/introduction/optimizers.rst

```diff
@@ -32,6 +32,8 @@ Some of these are specific to quantum optimization, such as the :mod:`~.QNGOptim
     ~pennylane.NesterovMomentumOptimizer
     ~pennylane.QNGOptimizer
     ~pennylane.RMSPropOptimizer
+    ~pennylane.RotosolveOptimizer
+    ~pennylane.RotoselectOptimizer
```

Comment:
Getting the following locally:
```
/pennylane/doc/code/api/pennylane.RotoselectOptimizer.rst: WARNING: document isn't included in any toctree
/pennylane/doc/code/api/pennylane.RotosolveOptimizer.rst: WARNING: document isn't included in any toctree
/pennylane/doc/code/api/pennylane.variable.VariableRef.rst: WARNING: document isn't included in any toctree
```
Though not sure where exactly this could be resolved.

Reply:
While building the docs? I don't seem to get any warnings at all. Not sure how that should be resolved either. As far as I can tell there's no difference between these new optimizers and the old ones with regards to documentation. But maybe I'm missing something.

Reply:
Yes, this slipped my mind, my fault! Thanks :)